shell/packages/accounts-email-token/token_server.js

```diff
@@ -83,25 +89,42 @@ var ROOT_URL = Url.parse(process.env.ROOT_URL);
 var HOSTNAME = ROOT_URL.hostname;
 
 var makeTokenUrl = function (email, token) {
-  return process.env.ROOT_URL + "/_emailToken/" + encodeURIComponent(email) + "/" + encodeURIComponent(token);
+  return process.env.ROOT_URL + "/_emailLogin/" + encodeURIComponent(email) + "/" + encodeURIComponent(token);
+};
+
+var makeTokenUrl = function (email, token, linkingIdentity) {
```

Comment:
This looks like it's redefining the var defined four lines earlier. Did you mean to delete the other one?